tools/logs2seq.py

```diff
@@ -0,0 +1,90 @@
+from typing import IO
+import requests
+import json
+import argparse
+import tarfile
+import logging
+from pathlib import Path
+import sys
+import os
+import boto3
+
+
+logger = logging.getLogger(Path(__file__).name)
+logging.basicConfig(
+    level=logging.INFO
+)
+
+VALID_LOG_FILES = [
+    'control-plane',
+    'compute-plane'
+]
+
+
+def is_valid_file(name: str) -> bool:
+    if any([name.find(log_file) != -1 for log_file in VALID_LOG_FILES]) and name.endswith(".log"):
+        return True
+    return False
```

Comment:
```suggestion
def is_valid_file(name: str) -> bool:
    return any((name.endswith(".log") and log_file in name for log_file in VALID_LOG_FILES]))
```
Several things : 
- any already return True or False, so you can return it directly instead of doing an if
- log_file in name is equivalent to name.find(log_file) != -1, but is more Pythonesque
- You can filter the .log inside the loop
- Not using a list comprehension [] but a generator expression () allows you to stop early. If the first log file matches, then true is returned instantaneously instead of doing all the list, then checking